source/validate.cpp

```diff
@@ -326,7 +328,7 @@ spv_result_t ValidateBinaryUsingContextAndValidationState(
     spv_instruction_t inst;
     spvInstructionCopy(&binary->code[index], static_cast<SpvOp>(opcode),
                        wordCount, endian, &inst);
-    instructions.push_back(inst);
+    instructions.emplace_back(std::move(inst));
```

Comment:
@atgoo, the issue here is the `std::vector` embedded in `inst`. I will take another look at it and undo this change if it's really losing.